jaas/src/main/java/com/griddynamics/jagger/jaas/rest/ReportingRestController.java

```diff
@@ -0,0 +1,50 @@
+package com.griddynamics.jagger.jaas.rest;
+
+import com.griddynamics.jagger.jaas.service.DynamicReportingService;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.context.request.async.WebAsyncTask;
+
+import java.io.IOException;
+
+/**
+ * JaaS REST API controller based on Spring MVC which generates Jagger test execution reports.
+ */
+@RequestMapping(value = "/report")
+@RestController
+public class ReportingRestController {
+    
+    private final DynamicReportingService dynamicReportingService;
+    
+    public ReportingRestController(DynamicReportingService dynamicReportingService) {
```

Comment:
Didn't you forget to @Autowire it? Or how do you inject DynamicReportingService?